apps/nuxt3/.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
Why this?

Reply:
Nuxt requires `shamefully-hoist` when using pnpm to build properly. Otherwise, it will cause dependency errors.

Reply:
This is currently tracked here; it can be removed once this issue is resolved: https://github.com/nuxt/nuxt/issues/14146